cmd/crd/cmd/generate.go

```diff
@@ -66,5 +66,8 @@ func GeneratorForFlags(f *flag.FlagSet) *crdgenerator.Generator {
 	// TODO: Do we need this? Is there a possibility that a crd is namespace scoped?
 	f.StringVar(&g.Namespace, "namespace", "", "CRD namespace, treat it as root scoped if not set")
 	f.BoolVar(&g.SkipMapValidation, "skip-map-validation", true, "if set to true, skip generating validation schema for map type in CRD.")
+	f.StringVar(&g.ApisPath, "apis-path", "pkg/apis", "the path to search for apis")
```

Comment:
nit: I generally prefer `API` instead of `Api` here.

Reply:
here and in method names (`s/setApisPkg/setAPIsPkg`) as well (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
done